machine/v1beta1/types_vsphereprovider.go

```diff
@@ -172,6 +179,15 @@ type NetworkDeviceSpec struct {
 	AddressesFromPools []AddressesFromPool `json:"addressesFromPools,omitempty"`
 }
 
+// VSphereDisk describes additional disks for vSphere.
+type VSphereDisk struct {
+	// The device name exposed. Used to identify the disk.
```

Comment:
Godoc should start with the serialised version.

What are the requirements of this string? Max/Min/Characters/Patterns?

Reply:
I updated godoc with these details.